index.d.ts

```diff
@@ -0,0 +1,14 @@
+import * as Handler from './types/handler';
+import * as Options from './types/options';
+
+export = errorHandler;
+
+declare function errorHandler(
```

Comment:
Let's call this `errorHandlerFactory` because it is a factory function returning back a middleware handler.